Tests/test_Restriction.py

```diff
@@ -8,8 +8,23 @@
 
 from Bio.Restriction import Analysis, Restriction, RestrictionBatch
 from Bio.Restriction import CommOnly, NonComm, AllEnzymes
-from Bio.Restriction import (Acc65I, Asp718I, BamHI, EcoRI, EcoRV, KpnI, SmaI,
-                             MluCI, McrI, NdeI, BsmBI, AanI, EarI, SnaI, SphI)
+from Bio.Restriction import (
+    Acc65I,
+    Asp718I,
+    BamHI,
+    EcoRI,
+    EcoRV,
+    KpnI,
+    SmaI,
+    MluCI,
+    McrI,
+    NdeI,
+    BsmBI,
+    AanI,
+    EarI,
+    SnaI,
+    SphI,
+)
```

Comment:
My personal preference would be:

```python
from Bio.Restriction import Acc65I
...
from Bio.Restriction import SphI
```

See discussion on #2828 - we have yet to agree any particular style, so this is fine for now.

Reply:
My personal preference would be to let it stand as it was, but let black do what black wants to do here.

Reply:
Yes, leave it as is for now - we can settle any preferred import style later on the linked issue.